core/client/fs/src/main/java/alluxio/client/file/BaseFileSystem.java

```diff
@@ -147,13 +148,15 @@ public boolean isClosed() {
   @Override
   public void createDirectory(AlluxioURI path)
       throws FileAlreadyExistsException, InvalidPathException, IOException, AlluxioException {
-    createDirectory(path, CreateDirectoryPOptions.getDefaultInstance());
+    createDirectory(path, FileSystemOptions.createDirectoryDefaults(mFsContext.getConf()));
```

Comment:
You don't need to pass the defaults since the options that you pass from here will be merged once again with the defaults. getDefaultInstance() should be fine in these utility versions.

Reply:
Same for all utility methods (the ones without options) in BaseFileSystem.

Reply:
Good catch, fixed